package org.api_sync.services.exceptions;

public class UsuarioNotFoundException extends RuntimeException {
    public UsuarioNotFoundException(Long usuarioId) {
        super("No se encontr√≥ el usuario con ID: " + usuarioId);
    }
} 